### PR TITLE
Update Chinese Translation - 2

### PR DESCRIPTION
**Pull request by IFRFSX**

### CODE REVIEW

**Comment by SmallJoker** [comment]
Is this PR complete, and ready for reviewing?

**Comment by IFRFSX** [comment]
you can start reviewing!

**Comment by SmallJoker** [comment]
I can only judge whether the translation placeholders (`@1`...) are used correctly. As long the translations fit into the in-game context I don't have any objections.

LGTM :+1:

**Comment by IFRFSX** [comment]
@SmallJoker 

ask a question:

When does minetest team plan to release version 5.2?

File: mods/default/locale/con.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
```
Comment (sfan5) [inline mods/default/locale/con.py]: While such scripts may be useful for your translation, please don't commit them to the repository.

Comment (IFRFSX) [reply mods/default/locale/con.py]: Extra files have been deleted.

**Pull request merged**